iommu_in_memory_queues.adoc

```diff
@@ -180,7 +180,10 @@ operand is valid. Setting `PSCV` to 1 is allowed only for `IOTINVAL.VMA`. The
 the translations associated with the host (i.e. those where the second-stage
 is Bare) are operated on. When `GV` is 0, the `GSCID` operand is ignored.
 When `AV` is 0, the `ADDR` operand is ignored. When `PSCV` operand is 0, the
-`PSCID` operand is ignored.
+`PSCID` operand is ignored. When the `AV` operand is set to 1, if the `ADDR`
+operand specifies an invalid address, the implementation may either complete the
+command with no effect or may complete the command using an alternate, yet
+`UNSPECIFIED`, legal value for the `ADDR` operand.
```

Comment:
When the `AV` operand is set to 1, if the `ADDR` operand specifies an invalid address, the command may or may not perform any invalidations.
note:
An implementation may either complete the command with no effect or may complete the command using an alternate, yet `UNSPECIFIED`, legal value for the `ADDR` operand. Note that  entries in general may be invalidated from the address translation cache at any time.